kotlin-data-wedge-lib/src/main/java/dk/gls/kdw/model/barcode/I2of5.kt

```diff
@@ -0,0 +1,71 @@
+package dk.gls.kdw.model.barcode
+
+import android.os.Bundle
+import dev.ahmedmourad.bundlizer.Bundlizer
+import dk.gls.kdw.model.barcode.model.SecurityLevel
+import kotlinx.serialization.SerialName
+import kotlinx.serialization.Serializable
+
+@Serializable
+data class I2of5(
+    @SerialName("decoder_i2of5")
+    val enabled: Boolean = true,
+
+    /**
+     * Sets the lower limit when decoding a barcode with a specific length range. For example, to decode a barcode containing between 4 and 12 characters, set this value to 4.
+     * Integer from 0–55
+     */
+    @SerialName("decoder_i2of5_length1")
+    val length1: Int,
+
+    /**
+     * Sets the upper limit when decoding a barcode with a specific length range. For example, to decode a barcode containing between 4 and 12 characters, set this value to 12.
+     * Integer from 0–55
+     */
+    @SerialName("decoder_i2of5_length2")
+    val length2: Int,
+
+    /**
+     * Sets the reader to read the barcode twice before accepting data.
+     */
+    @SerialName("decoder_i2of5_redundancy")
+    val redundancy: Boolean,
+
+    /**
+     * 	Transmits Interleaved 2 of 5 data with the check digit when enabled.
+     */
+    @SerialName("decoder_code11_report_check_digit")
```

Comment:
Is this the correct decoder serial name?

Reply:
Seems like a weird name, but yes according to the documentation.